### ISSUE TITLE
RNN with dropout / recurrent dropout and 1 timestep shows warning with CNTK

### ISSUE THREAD

**Issue by bmigette** [keras-team/keras#9336]
Hello,

I am creating the following network:
```
model = Sequential()
model.add(LSTM(512,return_sequences=True,recurrent_dropout=0.2, 
              input_shape=(train_x_lstm.shape[1], train_x_lstm.shape[2])
         ))
model.add(Dropout(0.2))
model.add(LSTM(128))
model.add(Dense(len(selected_y_cols)))
model.add(Activation('sigmoid'))
model.compile(loss='binary_crossentropy', optimizer='rmsprop', metrics =["accuracy"])

history = model.fit(train_x_lstm, 
                    train_y_lstm[selected_y_cols].values,
                    epochs=50,batch_size=64 ,
                    validation_data=(validate_x_lstm, validate_y_lstm[selected_y_cols].values), 
                    verbose=1, shuffle=False)
```
When using CNTK as backend, I am getting error:
'RNN dropout is not supported with the CNTK backend when using dynamic RNNs (i.e. non-unrolled).  You can either set `unroll=True`, set `dropout` and `recurrent_dropout` to 0, 'or use a different backend.'

However, I cannot unroll, because train_x_lstm.shape[1] = 1

**Comment by bmigette** [comment]
Precision, this is the error when attempting to unrolling:

ValueError: Cannot unroll a RNN if the time dimension is undefined or equal to 1.

**Comment by bmigette** [comment]
Actually this is fixed in 2.1.3... I updated before posting but looks like I forgot to restart python session or some cleared some cache. Closing issue.
